simulation_parameters/microbe_stage/organelles.json

```diff
@@ -22,6 +22,14 @@
     "displayScene": "res://assets/models/organelles/Pilus.tscn",
     "name": "ORGANELLE_PILUS",
     "iconPath": "res://assets/textures/gui/bevel/parts/PilusIcon.png",
+    "unlockConditions": [
+      {
+        "EngulfedMicrobesAbove": 5
```

Comment:
You've correctly seen the new style for our JSON files here, but this file is old so it uses camelCase. To maintain file uniformity and make it less surprising to read this file, could you convert the parts you added to camelCase (`engulfedMicrobesAbove`)?